projects/plugins/backup/src/js/components/Admin.js

```diff
@@ -28,17 +32,58 @@ const Admin = () => {
 	};
 
 	return (
-		<div id="jetpack-backup-admin-container" className="wrap">
-			<h1>Jetpack Backup Plugin</h1>
-			{ ! isUserConnected && renderPromptForConnection() }
-			{ isUserConnected && isRegistered && (
-				<p className="notice notice-success">
-					{ __(
-						'Site and User Connected. Todo: Show Backup plugin centric data :)',
-						'jetpack-backup'
-					) }
-				</p>
-			) }
+		<div id="jetpack-backup-admin-container" className="jp-content">
+			<div className="jp-header">
+				<h1>Jetpack Backup Plugin - Placeholder Header</h1>
+			</div>
+			<div className="content">
+				{ ! isUserConnected && renderPromptForConnection() }
+				{ isUserConnected && isRegistered && (
+					<div>
+						<div className="jp-hero">
+							<Backups apiRoot={ APIRoot } apiNonce={ APINonce } />
+						</div>
+						<div className="jp-section">
+							<div className="jp-wrap">
+								<div className="jp-row">
+									<div class="lg-col-span-6 md-col-span-4 sm-col-span-4">
+										<h2>Where are my backups stored?</h2>
```

Comment:
Do you think we can start using `@wordpress/i18n` from the get-go for all those strings?

Reply:
Updated to use `i18n` calls.